docs/query_engine/config/annotated_config.yaml

```diff
@@ -79,3 +79,27 @@ resultOptions:
 # for quick local setup (which this config will send data to).
 tracing:
   backend: jaeger
+
+# limits place logical limitations on the resource usage of the query service,
+# expressed primarily in the number of datapoints used by a query.
+limits:
+  # perQuery limits apply to each individual call to the query service, and
+  # are evaluated in isolation.
+  perQuery:
+    # maxFetchedDatapoints limits the number of datapoints used in the
+    # course of a query. They are evaluated across all time series in the query;
+    # roughly, the number of datapoints for a
+    # query = num_series * resolution * length_of_query
+    # TODO (andrewmains12): the numbers here are not representative of
```

Comment:
+1 for the followup, would be good to quantify what this limit means wrt to memory usage